gradle.properties

```diff
@@ -116,7 +116,7 @@ artifactRedirecting.androidx.annotation.version=1.9.1
 artifactRedirecting.androidx.graphics.version=1.0.1
```

Comment:
Did you delete the previous folder before applying the AOSP state?

Reply:
yes